plugin/pkg/auth/authorizer/node/node_authorizer_test.go

```diff
@@ -732,6 +735,23 @@ func BenchmarkAuthorization(b *testing.B) {
 		},
 	}
 
+	podToAdd := &corev1.Pod{
+		ObjectMeta: metav1.ObjectMeta{Name: "testwrite", Namespace: "ns0"},
+		Spec: corev1.PodSpec{
+			NodeName:           "node0",
+			ServiceAccountName: "default",
+		},
+	}
+	for i := 0; i < opts.sharedSecretsPerPod; i++ {
```

Comment:
should we populate all the `*PerPod` dimensions here? could extract the pod-specific bits of `generate()` to `generatePod(node, namespace, pod string, opts sampleDataOpts) (*corev1.Pod, []*corev1.PersistentVolume, []*storagev1.VolumeAttachment)` and call that directly here to construct the pod

Reply:
done